cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -154,11 +154,8 @@ const deleteCertificate = async function (arn, region, hostedZoneId) {
             inUseByResources = Certificate.InUseBy || [];
             dnsValidationRecord = Certificate.DomainValidationOptions || [];
             if (inUseByResources.length) {
-                // Exponential backoff with jitter based on 200ms base
-                // component of backoff fixed to ensure minimum total wait time on
-                // slow targets.
-                const base = Math.pow(2, attempt);
-                await sleep(random() * base * 50 + base * 150);
+                // Deleting resources can be quite slow - so just sleep 30 seconds between checks.
```

Comment:
What magnitude are we thinking about?

Reply:
like, it failed after 3 minutes, but was deleted after 4. So i'm not sure how consistent it is or what.